### ISSUE TITLE
textures and style

### ISSUE THREAD

**Issue by tobyplowy** [LordStephen77/Minetest2D#3]
@LordStephen77 here we talk about textures and how the games graphics can be derected and improved

I have a question for you how much control do I have over this (graphic derection)

**Comment by tobyplowy** [comment]
@LordStephen77 post the textures Whenever you're ready 

Remember Give me a shout out with @tobyplowy so I get an email when you do

**Comment by LordStephen77** [comment]
@tobyplowy The first textures I use here are a bit public, so you can see the design I have in mind. All the textures I use are taken by OpenGameArt with Mit and GPL licenses.

**Comment by tobyplowy** [comment]
@LordStephen77 Ok I see

**Comment by tobyplowy** [comment]
@LordStephen77 this topic is also about Graphics so I want to ask are you planning to add some Advanced features like Dynamic Lighting and shading and particle effects? (I know this is for long term and not for any time soon)

**Comment by tobyplowy** [comment]
@LordStephen77 I can make a new player image but I'm not good at animation :/

**Comment by kingoscargames** [comment]
is the image made with "paint.net"? because it has translucent parts and i saw a "pdn.pdn" file in "vanilla_plus"

**Comment by kingoscargames** [comment]
maybe i could make a player image?

**Comment by LordStephen77** [comment]
@tobyplowy Yes, dynamic lighting and particle effects will be present. The use of the particle system fascinates me a lot and this project will make extensive use of these features. Same thing for the night and day cycle.

@kingoscargames Do not worry about what you see now. In a few hours this project is fully supported. All the resources will be made by me, and others with open licenses.

The player has redesigned it, then we decide how to improve it, one step at a time. Now I work on the code for the ground generation algorithm (perlin) and for the night and day cycle.

**Comment by kingoscargames** [comment]
ok

**Comment by kingoscargames** [comment]
then i will finish this image for myself

**Comment by kingoscargames** [comment]
the same image but with more detailed legs and arms:
<img width="60" alt="player 3x" src="https://user-images.githubusercontent.com/29333817/27036925-ba6e6d6e-4f86-11e7-8e76-2c51074dc652.png">

**Comment by tobyplowy** [comment]
@kingoscargames please give LordStephen77 some time to release the first build!

**Comment by kingoscargames** [comment]
OK

**Comment by tobyplowy** [comment]
@LordStephen77 Can you remove some of the comments on this issue (there so much junk and so many off Topic comments)

**Comment by kingoscargames** [comment]
i just posted the image for if anyone wants it

**Comment by tobyplowy** [comment]
@kingoscargames please Oskar remove your comments and post them in a new issue
I want to keep this issue as clean and as professional as possible (I hope you understand)

**Comment by kingoscargames** [comment]
only the image is still here (and this)

**Comment by tobyplowy** [comment]
@kingoscargames Can you remove these 2 comments also (btw I got rid of my useles comments as well

> only the image is still here (and this)

> maybe i could make a player image?

I will remove this comment after you removed the other comments
:)

**Comment by kingoscargames** [comment]
i did
